examples/lstm_stateful_seq.py

```diff
@@ -0,0 +1,102 @@
+'''Example script to predict sequence using stateful rnns.
+The sequence predicted is a cosine curve with the amplitude
+being exponentially diminished.
+At least 10 epochs are required before the generated text
+starts sounding coherent.
+'''
+
+import numpy as np
+from keras.models import Sequential
+from keras.layers.core import Dense
+from keras.layers.recurrent import LSTM
+
+
+# since we are using stateful rnn tsteps can be set to 1
+tsteps = 1
+batch_size = 25
+epochs = 15
+# number of elements ahead that are used to make the prediction
+lahead = 1
+
+
+def genCosineAmp(amp=100, period=25, x0=0, xn=50000, step=1, k=0.0001):
```

Comment:
Hi,
Keras uses lower_case_underscores for function and variables names (genCosineAmp , also expectedOutput -> expected_output)